menu.c

```diff
@@ -32,8 +32,12 @@
 #include <string.h>
 #include <stdbool.h>
 
+#if defined(HAVE_LIBREADLINE) && HAVE_LIBREADLINE
```

Comment:
Would be also okay, yes. If it would be not defined, then configure would complain.